Golubeva/lab3/src/l3.py

```diff
@@ -0,0 +1,118 @@
+class Graph:
+
+    def __init__(self, graph):
+        self.graph = graph
+        self.row = len(graph)
+
+    def searchBFS(self, source, outlet, parent):
+
+        visited = self.row * [False] #list
```

Comment:
Как будто так понятнее
![image](https://user-images.githubusercontent.com/42893255/112961001-a31c9200-914d-11eb-9cd8-d097e26170ae.png)
Вот прмиер со строками
```suggestion
        visited =  [False] * self.row #list
```